### PR TITLE
Czech translations

### PR DESCRIPTION
**Pull request by YetheSamartaka**
- added all missing Czech Translations
- resolved issue with Turkish duplicity(core.xml)

### CODE REVIEW

**Comment by ilbinek** [comment]
Thank you for the translations. This will certainly make using the addon way easier for Czechs and Slovaks who are not skilled in English.
There are a "few" typos/duplicates and suggestions, which I would be happy if you could take a look at.
Mainly the "SR" - "KV" and "LR" - "DV" ones.

And one last thing, "intercom" could be replaced with the Czech version "interkom".

Translating is hard and takes a lot of time, a lot of mistakes get through because your brain starts to ignore them the more you read the text. There's a high chance I missed some too, so I encourage others to give it another look.

**Comment by YetheSamartaka** [comment]
> 

Thank you for pointing it out, I will take a look at it this upcoming week.

**Comment by YetheSamartaka** [comment]
> SR

I have changed the intercom, SR and KV to fit the Czech language better. Is there anything else I should take a look at?

**Review by ilbinek — CHANGES_REQUESTED** [review]
Oh, I forgot to submit the review. My bad.
There are a few other things I found.

File: addons/core/stringtable.xml [hunk 1]
```diff
@@ -34,6 +37,7 @@
                 <Russian>TFAR-Устарело</Russian>
                 <Turkish>TFAR Tarafından Kullanımdan Kaldırıldı</Turkish>
                 <Italian>TFAR Deprecato</Italian>
+                <Czech>TFAR-zastaralé</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:40]: Instead I removed the "-" to better fit Czech grammar.

File: addons/core/stringtable.xml [hunk 2]
```diff
@@ -140,6 +145,7 @@
                 <Russian>TFAR - Настройки клиента</Russian>
                 <Turkish>TFAR - Kişisel ayarlar</Turkish>
                 <Italian>TFAR - Impostazioni lato client</Italian>
+                <Czech>TFAR – Nastavení na straně klienta</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:148]: Although my version is longer, it gives a more accurate translation of what that setting does.

File: addons/core/stringtable.xml [hunk 3]
```diff
@@ -728,36 +786,41 @@
                 <Chinese>使用宙斯編輯器時可以聆聽到玩家的語音</Chinese>
                 <Chinesesimp>使用宙斯编辑器时可以聆听到玩家的语音</Chinesesimp>
                 <Italian>Ascolta la voce dalla fotocamera del curatore quando sei nell'interfaccia di Zeus</Italian>
+                <Czech>Slyšet hlasy z kamery kurátory, když jste v rozhraní Dia</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:789]: That was just a typo. Fixed.

File: addons/core/stringtable.xml [hunk 4]
```diff
@@ -728,36 +786,41 @@
...
                 <English>Hear voice from Splendid Camera when in Splendid Camera interface</English>
+                <Czech>V rozhraní Splendid Kamery uslyšíte hlas ze Splendid kamery</Czech>
             </Key>
-
             <Key ID="STR_TFAR_SETTINGS_noAutomoveSpectator">
                 <English>No Automove in Spectator</English>
                 <German>Kein Automove im Spectator</German>
                 <Japanese>スペクテイターで自動移動を無効化</Japanese>
                 <Italian>Nessun movimento automatico in spettatore</Italian>
+                <Czech>Žádný automatický pohyb v režimu diváka</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:804]: "Automaticky" is quite a keyword in this case, so I will leave it included there.

File: addons/core/stringtable.xml [hunk 5]
```diff
@@ -953,6 +1036,7 @@
                 <Korean>기본 개인 무전 설정 (CSAT)</Korean>
                 <Russian>Стандартная рация солдата (CSAT)</Russian>
                 <Italian>Radio-fuciliere predefinita (CSAT)</Italian>
+                <Czech>Výchozí rádi střelce (CSAT)</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:1039]: Typo. Fixed.

File: addons/core/stringtable.xml [hunk 6]
```diff
@@ -1302,28 +1417,33 @@
...
                 <English>Disable Teamspeak Mute</English>
                 <German>Deaktiviere Teamspeak Mute</German>
                 <Italian>Disabilita Teamspeak Mute</Italian>
+                <Czech>Zakázat ztlumení Teamspeaku</Czech>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_disableAutoMute_desc">
                 <English>Disables automatically local muting players in Teamspeak. (causes high Performance cost)</English>
                 <German>Deaktiviert automatisches Lokales Muten von Spielern im Teamspeak. (Erzeugt hohe Rechenlast)</German>
                 <Italian>Disabilitato automaticamente l'audio locale dei giocatori in Teamspeak. (causa un alto costo delle prestazioni)</Italian>
+                <Czech>Zakáže automatické ztlumení ztlumení hráčů v Teamspeaku. (vysoce náročné na výkon)</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:1446]: Fixed.

File: addons/core/stringtable.xml [hunk 7]
```diff
@@ -2213,6 +2357,7 @@ Normal radyo su altı radyosuyla iletişim kuramaz (herhangi bir ses gelmez). Da
                 <Chinesesimp>戴上耳机</Chinesesimp>
                 <Russian>Надеть гарнитуру</Russian>
                 <Italian>Alza auricolare</Italian>
+                <Czech>Zvýšit headset</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:2360]: Improved.

File: addons/core/stringtable.xml [hunk 8]
```diff
@@ -2223,6 +2368,7 @@ Normal radyo su altı radyosuyla iletişim kuramaz (herhangi bir ses gelmez). Da
                 <Chinesesimp>戴上耳机 / 取下耳机</Chinesesimp>
                 <Russian>Снять/Надеть гарнитуру</Russian>
                 <Italian>Alza/abbassa l'auricolare</Italian>
+                <Czech>Zvýšit/snížit headset</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:2371]: Improved.

File: addons/core/stringtable.xml [hunk 9]
```diff
@@ -3299,6 +3451,7 @@ Normal radyo su altı radyosuyla iletişim kuramaz (herhangi bir ses gelmez). Da
             <Chinesesimp>TFAR - 无法从短程无线电复制设定到长程无线电上</Chinesesimp>
             <Russian>TFAR - невозможно скопировать из БС в ДС</Russian>
             <Italian>TFAR - impossibile copiare da SR a LR</Italian>
+            <Czech>TFAR - nelze kopírovat ze SR do LR</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml]: Fixed.

File: addons/core/stringtable.xml [hunk 10]
```diff
@@ -2203,6 +2346,7 @@ Normal radyo su altı radyosuyla iletişim kuramaz (herhangi bir ses gelmez). Da
                 <Chinesesimp>取下耳机</Chinesesimp>
                 <Russian>Снять гарнитуру</Russian>
                 <Italian>Auricolare inferiore</Italian>
+                <Czech>Snížit headset</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:2349]: Fixed.

File: addons/external_intercom/stringtable.xml [hunk 11]
```diff
@@ -4,109 +4,135 @@
...
         <Key ID="STR_TFAR_EXTERNAL_INTERCOM_PHONE_BUSY">
             <Original>Intercom Phone in use</Original>
             <English>Intercom Phone in use</English>
+            <Czech>Intercom telefon je používán</Czech>
         </Key>
-        <Key ID="STR_TFAR_EXTERNAL_INTERCOM_SOMEONE_USING_PHONE">
+		<Key ID="STR_TFAR_EXTERNAL_INTERCOM_SOMEONE_USING_PHONE">
             <Original><br /><br /><t align='center'>%1 is currently using the Intercom Phone</t></Original>
             <English><br /><br /><t align='center'>%1 is currently using the Intercom Phone</t></English>
+			<Czech><br /><br /><t align='center'>%1 is momentálně používá telefon intercomu</t></Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/external_intercom/stringtable.xml]: Fixed.

File: addons/external_intercom/stringtable.xml [hunk 12]
```diff
@@ -4,109 +4,135 @@
...
         </Key>
         <Key ID="STR_TFAR_EXTERNAL_INTERCOM_PICKUP_PHONE">
             <Original>Pickup Intercom Phone</Original>
             <English>Pickup Intercom Phone</English>
+            <Czech>Vzít intercom telefon</Czech>
         </Key>
         <Key ID="STR_TFAR_EXTERNAL_INTERCOM_PUT_AWAY_PHONE">
             <Original>Put away Intercom Phone</Original>
             <English>Put away Intercom Phone</English>
+            <Czech>Dát pryč intercom telefon</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/external_intercom/stringtable.xml]: Agreed.

File: addons/external_intercom/stringtable.xml [hunk 13]
```diff
@@ -4,109 +4,135 @@
...
         <Container name="settings">
             <Key ID="STR_TFAR_EXTERNAL_INTERCOM_SETTINGS_CATEGORY_EXTERNAL_INTERCOM">
                 <Original>External Intercom</Original>
                 <English>External Intercom</English>
+                <Czech>Vnější intercom</Czech>
             </Key>
             <Key ID="STR_TFAR_EXTERNAL_INTERCOM_SETTING_WIRELESS_HEADGEAR_HEADER">
                 <Original>Additional Wireless Compatible Headgear</Original>
                 <English>Additional Wireless Compatible Headgear</English>
+                <Czech>Další vybavení compatibilní s bezdrátem</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/external_intercom/stringtable.xml:63]: Improved.

File: addons/external_intercom/stringtable.xml [hunk 14]
```diff
@@ -4,109 +4,135 @@
...
             </Key>
             <Key ID="STR_TFAR_EXTERNAL_INTERCOM_SETTING_WIRELESS_HEADGEAR_DESC">
                 <Original>A list of Headgear, in addition to the base list that are able to be connected to a vehicle's intercom system wirelessly</Original>
                 <English>A list of Headgear, in addition to the base list that are able to be connected to a vehicle's intercom system wirelessly</English>
+                <Czech>Seznam vybavení, které rozšíří základní seznam vybavení schopnýchse bezdrátově připojit k intercomu vozidla</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/external_intercom/stringtable.xml]: Fixed.

File: addons/external_intercom/stringtable.xml [hunk 15]
```diff
@@ -4,109 +4,135 @@
...
-        <Key ID="STR_TFAR_EXTERNAL_INTERCOM_SOMEONE_USING_PHONE">
+		<Key ID="STR_TFAR_EXTERNAL_INTERCOM_SOMEONE_USING_PHONE">
             <Original><br /><br /><t align='center'>%1 is currently using the Intercom Phone</t></Original>
             <English><br /><br /><t align='center'>%1 is currently using the Intercom Phone</t></English>
+			<Czech><br /><br /><t align='center'>%1 is momentálně používá telefon intercomu</t></Czech>
         </Key>
         <Key ID="STR_TFAR_EXTERNAL_INTERCOM_HEADGEAR_NOT_WIRELESS_CAPABLE">
             <Original><br /><br /><t align='center'>Your %1 is not Wireless Intercom capable</t></Original>
             <English><br /><br /><t align='center'>Your %1 is not Wireless Intercom capable</t></English>
+			<Czech><br /><br /><t align='center'>Your %1 není schopno bezdrátového spojení</t></Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/external_intercom/stringtable.xml:42]: Fixed.

File: addons/core/stringtable.xml [hunk 16]
```diff
@@ -791,6 +856,7 @@
                 <Korean>기본 단파 설정</Korean>
                 <Russian>Частоты БС по умолчанию:</Russian>
                 <Italian>Frequenze standard a corto raggio:</Italian>
+                <Czech>Výchozí SR frekvence:</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml]: Indeed.

File: addons/core/stringtable.xml [hunk 17]
```diff
@@ -526,6 +563,7 @@
                 <Russian>Задержка при отпускании тангенты (мсек)</Russian>
                 <Turkish>milisaniye cinsinden tangentReleaseDelay</Turkish>
                 <Italian>tangentReleaseDelay in millisecondi</Italian>
+                <Czech>tangentReleaseDelay v milisekundách</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/core/stringtable.xml:566]: Copy that.

File: addons/handhelds/stringtable.xml [hunk 18]
```diff
@@ -109,6 +122,7 @@
             <Chinesesimp>AN/PRC-148 JEM短程个人无线电，有效距离5公里</Chinesesimp>
             <Turkish>AN/PRC-148 JEM kişisel telsiz 5km</Turkish>
             <Italian>AN/PRC-148 JEM radio personale 5 km</Italian>
+            <Czech>AN/PRC-148 JEM  Osobní rádio 5km</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/handhelds/stringtable.xml:125]: Fixed.

File: addons/handhelds/stringtable.xml [hunk 19]
```diff
@@ -94,11 +105,13 @@
             <Chinesesimp>AN/PRC-152短程个人无线电，有效距离5公里</Chinesesimp>
             <Turkish>AN/PRC-152 kişisel telsiz 5km</Turkish>
             <Italian>AN/PRC-152 radio personale 5 km</Italian>
+            <Czech>AN/PRC-152  Osobní rádio 5km</Czech>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/handhelds/stringtable.xml:108]: Fixed.

**Comment by YetheSamartaka** [comment]
> 

I've gone through all you've suggested, improved and fixed everything. Thank you for reading it all through and pointing out things to fix/improve.

**Review by ilbinek — APPROVED** [review]
A little necro, but all looks good to me.

**Comment by dedmen** [comment]
Thank you for the reviewing, I don't have the time 😢

**Pull request merged**